tests/flags.rs

```diff
@@ -87,6 +87,24 @@ fn flag_using_long() {
     assert!(m.is_present("color"));
 }
 
+#[cfg(feature = "suggestions")]
+#[test]
+fn flag_using_long_with_literals() {
+    use clap::ErrorKind;
+
+    let m = App::new("flag")
+        .args(&[Arg::from("--rainbow 'yet another flag'").multiple_occurrences(true)])
+        .try_get_matches_from(vec![
+            "",
+            "--rainbow",
+            "--rainbow",
+            "--rainbow=false",
+            "--rainbow=true",
```

Comment:
Update this to just `vec!["--rainbow=false"]`.

Reply:
I guess you meant `vec!["", "--rainbow=false"]`.